layouts/partials/head.html

```diff
@@ -54,4 +54,4 @@
 
     <!-- Twitter Cards -->
     {{ template "_internal/twitter_cards.html" . }}
-</head>
\ No newline at end of file
+</head>
```

Comment:
Mind if I do a PR where for all files to add a newline? This way git wont trigger on the last lines like it does here :)

Reply:
No, I don't mind at all. :)